bedrock/pocket/templates/pocket/license-ios.html

```diff
@@ -0,0 +1,2178 @@
+          WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+          implied, including, without limitation, any warranties or conditions
+          of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A
+          PARTICULAR PURPOSE. You are solely responsible for determining the
+          appropriateness of using or redistributing the Work and assume any
+          risks associated with Your exercise of permissions under this License.
+        </p>
+      </li>
+      <li>
+        <p><strong>Limitation of Liability</strong>.</p>
+        <p>In no event and under no legal theory,
+          whether in tort (including negligence), contract, or otherwise,
+          unless required by applicable law (such as deliberate and grossly
+          negligent acts) or agreed to in writing, shall any Contributor be
+          liable to You for damages, including any direct, indirect, special,
+          incidental, or consequential damages of any character arising as a
+          result of this License or out of the use or inability to use the
+          Work (including but not limited to damages for loss of goodwill,
+          work stoppage, computer failure or malfunction, or any and all
+          other commercial damages or losses), even if such Contributor
+          has been advised of the possibility of such damages.
+        </p>
+      </li>
+      <li>
+        <p><strong>Accepting Warranty or Additional Liability</strong>.</p>
+        <p>While redistributing
+          the Work or Derivative Works thereof, You may choose to offer,
+          and charge a fee for, acceptance of support, warranty, indemnity,
+          or other liability obligations and/or rights consistent with this
+          License. However, in accepting such obligations, You may act only
+          on Your own behalf and on Your sole responsibility, not on behalf
+          of any other Contributor, and only if You agree to indemnify,
+          defend, and hold each Contributor harmless for any liability
+          incurred by, or claims asserted against, such Contributor by reason
+          of your accepting any such warranty or additional liability.
+        </p>
+      </li>
+    </ol>
+    <p>END OF TERMS AND CONDITIONS</p>
+    <p>APPENDIX: How to apply the Apache License to your work.</p>
+    <p>To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don&#39;t include
+      the brackets!) The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+    </p>
+    <p>Copyright [yyyy] [name of copyright owner]</p>
```

Comment:
@bassrock For some of the Apache licenses, the copyright section (noted here) don't have the year and name of copyright owner. I also checked some of the packages, and they have this format w/ missing information. What should we do in this context?

Reply:
@reemhamz if the source is missing it I think we keep it as is.